src/schemas/21-686C/schema.js

```diff
@@ -464,72 +572,27 @@ let schema = {
           disabled: {
             type: 'boolean'
           },
-          married: {
+          previouslyMarried: {
             type: 'boolean'
           },
-          previouslyMarried: {
+          childInHousehold: {
             type: 'boolean'
           },
-          childPlaceOfBirth: {
+          childAddress: {
             type: 'object',
-            oneOf: [
-              {
-                required: ['childCountryOfBirthDropdown', 'childCityOfBirth', 'childStateOfBirth'],
-                properties: {
-                  childCountryOfBirthDropdown: {
-                    type: 'string',
-                    'enum': [countryUSA],
-                    default: countryUSA
-                  },
-                  childCityOfBirth: {
-                    type: 'string',
-                    maxLength: 30,
-                    minLength: 1,
-                    pattern: textRegex
-                  },
-                  childStateOfBirth: {
-                    type: 'string',
-                    maxLength: 50,
-                    'enum': states.map(state => state.value),
-                    enumNames: states.map(state => state.label)
-                  }
-                },
-                additionalProperties: false
-              },
-              {
-                required: ['childCountryOfBirthDropdown', 'childCountryOfBirthText'],
-                properties: {
-                  childCountryOfBirthDropdown: {
-                    type: 'string',
-                    'enum': [countryNotInList],
-                    default: countryNotInList
-                  },
-                  childCountryOfBirthText: {
-                    type: 'string',
-                    maxLength: 50,
-                    minLength: 1,
-                    pattern: textRegex
-                  }
-                },
-                additionalProperties: false
-              },
-              {
-                required: ['childCountryOfBirthDropdown'],
-                properties: {
-                  childCountryOfBirthDropdown: {
-                    type: 'string',
-                    'enum': countriesForeign
-                  }
-                },
-                additionalProperties: false
-              }
-            ]
-          }
+            oneOf: addressDefs
+          },
+          personWhoLivesWithChild: schemaHelpers.getDefinition('fullName')
         }
       }
     }
   },
-  required: ['privacyAgreementAccepted']
+  required: [
+    'privacyAgreementAccepted',
+    'veteranFullName',
+    'veteranAddress',
+    'maritalStatus'
+  ]
```

Comment:
How should we handle requirement of `ssn` and/or `file number`?

Reply:
https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/src/schemas/21P-527EZ/schema.js#L18

Reply:
Thanks